samples/AnomalyDetector/src/Generated/Docs/Multivariate.xml

```diff
@@ -5,190 +5,82 @@
 Console.WriteLine(result.GetProperty("summary").GetProperty("setupInfo").GetProperty("topContributorCount").ToString());
 Console.WriteLine(result.GetProperty("summary").GetProperty("setupInfo").GetProperty("startTime").ToString());
 Console.WriteLine(result.GetProperty("summary").GetProperty("setupInfo").GetProperty("endTime").ToString());
 Console.WriteLine(result.GetProperty("results")[0].GetProperty("timestamp").ToString());
-Console.WriteLine(result.GetProperty("results")[0].GetProperty("value").GetProperty("isAnomaly").ToString());
-Console.WriteLine(result.GetProperty("results")[0].GetProperty("value").GetProperty("severity").ToString());
-Console.WriteLine(result.GetProperty("results")[0].GetProperty("value").GetProperty("score").ToString());
-Console.WriteLine(result.GetProperty("results")[0].GetProperty("value").GetProperty("interpretation")[0].GetProperty("variable").ToString());
-Console.WriteLine(result.GetProperty("results")[0].GetProperty("value").GetProperty("interpretation")[0].GetProperty("contributionScore").ToString());
-Console.WriteLine(result.GetProperty("results")[0].GetProperty("value").GetProperty("interpretation")[0].GetProperty("correlationChanges").GetProperty("changedVariables")[0].ToString());
-Console.WriteLine(result.GetProperty("results")[0].GetProperty("errors")[0].GetProperty("code").ToString());
-Console.WriteLine(result.GetProperty("results")[0].GetProperty("errors")[0].GetProperty("message").ToString());
 ]]></code></example>
     </member>
     <member name="TrainMultivariateModelAsync(ModelInfo,CancellationToken)">
       <example>
 This sample shows how to call TrainMultivariateModelAsync.
 <code><![CDATA[
-Uri endpoint = new Uri("<https://my-service.azure.com>");
-AzureKeyCredential credential = new AzureKeyCredential("<key>");
-Multivariate client = new AnomalyDetectorClient(endpoint, credential).GetMultivariateClient();
-
-ModelInfo modelInfo = new ModelInfo("<dataSource>", DateTimeOffset.Parse("2022-05-10T18:57:31.2311892Z"), DateTimeOffset.Parse("2022-05-10T18:57:31.2311892Z"));
-Response<AnomalyDetectionModel> response = await client.TrainMultivariateModelAsync(modelInfo);
-]]></code>
-This sample shows how to call TrainMultivariateModelAsync with all parameters.
-<code><![CDATA[
-Uri endpoint = new Uri("<https://my-service.azure.com>");
+Uri endpoint = new Uri("<endpoint>");
 AzureKeyCredential credential = new AzureKeyCredential("<key>");
 Multivariate client = new AnomalyDetectorClient(endpoint, credential).GetMultivariateClient();
 
-ModelInfo modelInfo = new ModelInfo("<dataSource>", DateTimeOffset.Parse("2022-05-10T18:57:31.2311892Z"), DateTimeOffset.Parse("2022-05-10T18:57:31.2311892Z"))
+ModelInfo modelInfo = new ModelInfo("https://multiadsample.blob.core.windows.net/data/sample_data_2_1000.csv", DateTimeOffset.Parse("2019-04-01T00:00:00Z"), DateTimeOffset.Parse("2019-04-02T00:00:00Z"))
```

Comment:
I expect there are a lot of changes of generated XML docs for protocol methods in regen, but I didn't find any change in in the regen PR, is this expected

Reply:
TCGC will not read the examples unless this configuration is set: https://github.com/Azure/azure-rest-api-specs/blob/ba268c5654f002bc17914e0bc157f19529f1fec1/specification/communication/Communication.JobRouter/tspconfig.yaml#L30

Reply:
I enabled one of them to have the real examples: https://github.com/Azure/azure-sdk-for-net/pull/45614